.github/workflows/build_and_test.yml

```diff
@@ -30,16 +30,6 @@ jobs:
       with:
         fetch-depth: '0'
     - run: git fetch --depth=1 origin +refs/tags/*:refs/tags/* +refs/heads/*:refs/remotes/origin/*
-    - name: Set up Python 3.8
-      if: github.event_name == 'push' || github.event_name == 'schedule'
-      uses: actions/setup-python@v4
-      with:
-        python-version: '3.8'
-    - name: Build and test (3.8)
-      if: github.event_name == 'push' || github.event_name == 'schedule'
-      shell: bash
-      run: |
-        ./.github/workflows/build-test nomypy
```

Comment:
Is this removing a test workflow? Should we replace 3.8 with 3.9 or 3.10, rather than deleting these lines?

Reply:
3.9 and 3.10 are both run on push and schedule, so I don't think that we are missing any tests execution with removing this lines?